cmd/kops-controller/main.go

```diff
@@ -179,16 +181,26 @@ func addNodeController(mgr manager.Manager, opt *config.Options) error {
 		return fmt.Errorf("identifier for cloud %q not implemented", opt.Cloud)
 	}
 
-	if opt.ConfigBase == "" {
-		return fmt.Errorf("must specify configBase")
-	}
+	if identifier != nil {
```

Comment:
Just a suggestion - the other way to structure this is to handle the "new" cases first, then fall-through to the other case.  I find it tends to be more intuitive to read.